tests/foreman/cli/test_satellitesync.py

```diff
@@ -383,6 +400,96 @@ def test_positive_complete_library_rh_repo(
             != ''
         )
 
+    @pytest.mark.tier3
+    @pytest.mark.upgrade
+    def test_positive_export_repository_docker(
+        self, target_sat, export_import_cleanup_module, module_org, docker_repo
+    ):
+        """Export docker repo via complete and incremental repository.
+
+        :id: 3c666ffd-d287-4006-b3a0-66d892fe4250
+
+        :setup:
+            1. Have a synchronized docker-type repo with immediate download policy.
+
+        :steps:
+            1. Export complete repository.
+            2. Export incremental repository.
+
+        :expectedresults:
+            1. Export path is created (with expected files) for complete export.
+            2. Export path is created (with expected files) for incremental export.
+
+        :BZ: 1650468
+
+        :customerscenario: true
+        """
+        # Verify export directory is empty
+        assert target_sat.validate_pulp_filepath(module_org, PULP_EXPORT_DIR) == ''
+        # Export complete and check the export directory
+        target_sat.cli.ContentExport.completeRepository({'id': docker_repo['id']})
+        assert '1.0' in target_sat.validate_pulp_filepath(module_org, PULP_EXPORT_DIR)
+        # Export incremental and check the export directory
+        target_sat.cli.ContentExport.incrementalRepository({'id': docker_repo['id']})
+        assert '2.0' in target_sat.validate_pulp_filepath(module_org, PULP_EXPORT_DIR)
+
+    @pytest.mark.tier3
+    @pytest.mark.upgrade
+    def test_positive_export_version_docker(
+        self, target_sat, export_import_cleanup_module, module_org, docker_repo
+    ):
+        """Export CV with docker repo via complete and incremental version.
+
+        :id: ddff4560-cd39-4ecc-a538-09aad9f64a73
+
+        :setup:
+            1. Have a synchronized docker-type repo with immediate download policy.
+
+        :steps:
+            1. Create a CV, add the docker repository and publish it.
+            2. Export complete version of the CV.
+            3. Publish new version of the CV.
+            4. Export incremental version of the CV.
+
+        :expectedresults:
+            1. Export path is created (with expected files) for complete CVV export.
+            2. Export path is created (with expected files) for incremental CVV export.
+
+        :BZ: 1650468
+
+        :customerscenario: true
+        """
+        # Create CV and publish
+        cv_name = gen_string('alpha')
+        cv = make_content_view({'name': cv_name, 'organization-id': module_org.id})
+        ContentView.add_repository(
+            {
+                'id': cv['id'],
+                'organization-id': module_org.id,
+                'repository-id': docker_repo['id'],
+            }
+        )
```

Comment:
can you update it to use either target_sat fixture for CLI property or old CLIs way? mixing both doesn't look good :D